docs/content/tutorials/kic/egress-walkthrough.md

```diff
@@ -132,14 +139,14 @@ The `target` application is a basic NGINX server listening on port 80. It return
 
 The `egress-driver` application is unable to reach the `target` Service because it is not injected with the sidecar proxy. We are running with `--mtls-mode=strict` which restricts the `egress-driver` to communicating using mTLS with other injected pods. As a result we cannot build traffic statistics for these requests.
 
-Now, let's use NGINX Plus Ingress Controller to create a secure internal route from the `egress-driver` application to the `target` Service.
+Now, let's use NGINX Ingress Controller to create a secure internal route from the `egress-driver` application to the `target` Service.
 
-### Install NGINX Plus Ingress Controller
+### Install NGINX Ingress Controller
 
-1. [Install the NGINX Plus Ingress Controller]( {{< ref "/tutorials/kic/deploy-with-kic.md#install-nginx-plus-ingress-controller-with-mtls-enabled">}} ). This tutorial will demonstrate installation as a Deployment.
+1. [Install the NGINX Ingress Controller]( {{< ref "/tutorials/kic/deploy-with-kic.md#install-nginx-plus-ingress-controller-with-mtls-enabled">}} ). This tutorial will demonstrate installation as a Deployment.
```

Comment:
Will need to update these links and their header names to not include "plus" anymore.

Reply:
Updated